src/gleam/map.gleam

```diff
@@ -97,7 +107,7 @@ if erlang {
 
 if javascript {
   fn do_from_list(list: List(#(k, v))) -> Map(k, v) {
-    list.fold(list, new(), insert_pair)
+    list_fold(list, new(), insert_pair)
```

Comment:
Let's implement this with regular recursion rather than copying the fold combinator into this module